kolibri_explore_plugin/assets/src/views/DiscoveryPageContenPacks.vue

```diff
@@ -122,6 +152,9 @@
       // channelLabel: "This week's channel: {channel}",
       careerLabel: 'Explore careers',
       curiousLabel: 'Feeling curious?',
+      aboutLabel: 'About the Endless Key',
```

Comment:
Nit: I think Heather prefers it without the article: "About Endless Key" . And the designs are always carring the error :) . Feel free to ask Heather.

Reply:
That makes sense! I went and changed it, we can double-check with Heather next week.